### PR TITLE
feat: Set up catalog app (sandbox)

### PR DESCRIPTION
**Pull request by borismelnik**
#5 

From Qt Creator:
1. Open in Qt Creator file `sandbox.pro`
2.  Build :)
3.  Run

From console:
1. run `path/to/qt/qmake sandbox.pro -spec macx-clang CONFIG+=debug CONFIG+=x86_64 && /usr/bin/make qmake_all`
2. run `make`
3. Binaries must be placed in `sandbox/bin`

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Great work!

File: sandbox/main.qml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            checkable: true
+            text: "Bars"
+        }
+        Button {
+            checkable: true
+            text: "Modals"
+        }
+        Button {
+            checkable: true
+            text: "inputs"
```
Comment (PascalPrecht) [inline sandbox/main.qml:50]: This should probably be "Inputs" then

File: sandbox/main.qml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        buttons: tabs.children
+    }
+
+    Flow {
+        id: tabs
+        width: parent.width
+        Button {
+            checkable: true
+            text: "Text Styles"
+        }
```
Comment (PascalPrecht) [inline sandbox/main.qml:23]: I don't believe it make sense to have this one as there most likely won't be multiple text style components (probably just `StatusText` which can then be adjusted in size and weight).

File: sandbox/main.qml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                scale: rootWindow.factor
+            }
+
+            Rectangle {
+                id: darkTheme
+
+                SplitView.fillWidth: true
+                SplitView.minimumWidth: rootWindow.width / 2
+                height: parent.height
+                color: "#2c2c2c"
```
Comment (PascalPrecht) [inline sandbox/main.qml:79]: Now that #23 has landed, we can make use of `Theme.palette.baseColor5` here.

**Review by PascalPrecht — CHANGES_REQUESTED** [review]

**Comment by PascalPrecht** [comment]
Really great work indeed!

I've noticed that, after running this, there's untracked `.qmake.stash` and `MakeFile` files.
Can you add them to `.gitignore` please?

**Review by PascalPrecht — APPROVED** [review]

**Comment by PascalPrecht** [comment]
This has landed as https://github.com/status-im/StatusQ/commit/3528b2ff440a8fb66609f7087f062643f5082132
(I've updated your commit message so it automatically closes #5)

Awesome work Boris!

**Pull request closed without merging**